### ISSUE TITLE
Sep 7 - 11: Rust Programming Basics

### ISSUE THREAD

**Issue by meaghanlewis** [microsoft/ReactorPlanning#46]
# Event Information
| Category | Details |
|-----------|---------|
| Reactor Topic | Coding, Languages, Frameworks |
| Monthly Topic (if applicable) | Skill Up September |
| Weekly Topic (if applicable) | Rust |
| Stream Date | Sep 7 - Sep 11, 2020 | 
| Stream Duration | 90 min | 
| Title | Rust Programming Basics|
| Description | Rust is a programming language that empowers developers to build reliable and efficient software. It is fun to write, easy to read, and high performing. In this workshop, we will explore how to start coding in Rust. This workshop will require you to use VS Code and the Rust extension to write code. |
| Content | https://github.com/microsoft/Reactors/tree/main/online-event-resources/web-development/rust101 |

Additional Resources:  
<Additional resources that can be linked from social media for promotion or added to the description of VOD>

# Workshop details
| Date |  Local Start Time |  Location |  Speaker | SharePoint ID |
|----- |------------------|-----------|----------|--------------|
| Sep 7 |  4:00 PM GMT+4 | Abu Dhabi | Sameer Arora | 7917 |
| Sep 7 |  6:00 PM  GMT+3 | Tel Aviv | Sameer Arora | 7918 |
| Sep 8 |  8:00 PM  GMT+10| Sydney | Andrew Smith | 7919 |
| Sep 8 |  6:30 PM GMT+5:30| Bangalore | Sameer Arora | 7920 |
| Sep 9 |  3:30 PM GMT-3| Sao Paulo | Tiago Guimarães | 7921 |
| Sep 9 |  7:30 PM GMT+8 | Shanghai | | 7922 |
| Sep 9 | 6:00 PM GMT+1  | London | Brian Sletten | 7923 |
| Sep 9 | 4:00 PM PDT | US/Canada | Andrew Smith | 7924 |

# Event Checklist

## 10 Weeks Prior
- [X] Reactor Content Team (RCT) opens a new issue for an upcoming local workshop filling in the Event Information and Workshop details tables. RCT member who owns content (content owner) this week will be the assignee.
- [X] RCT and content owner discuss details of the event in the comments of this issue. RCT adds a _confirmed-event_ label and _local-digital-event_ tag to the issue and adds event to the __Local Digital Events__ project. Issue will appear in the __Upcoming Streams__ column
- [X] RCT adds a _ready-for-marketing-edit_ label so the RCT can review title and description

## 8 Weeks Prior
- [x] RCT does a review of title and description and removes the _ready-for-marketing-edit_ label 
- [x] RCT moves the issue to the __Ready for SharePoint__ column
- [x] RCT informs the Reactor Marketing Team of the new event information
- [x] RCT confirms the speaker
- [x] RCT provides speaker with assets (e.g. slides, overlays) and the survey data for the event
- [x] RCT moves the issue to the __Content Preparation In Progress__ column

## 5 weeks Prior
- [ ] Content owner opens a PR with the new content for the workshop on the [customer-facing Reactor repository](https://github.com/microsoft/Reactors) and adds the _content-development_ and _local-digital-event_ tag to the PR. 
- [ ] Content owner adds to the same PR modification to the [Streaming README](https://github.com/microsoft/Reactors/tree/main/Online) 
- [ ] RCT reviews, approves, and merges the changes
- [ ] RCT shares content with the vendor/instructor
- [ ] RCT provides vendor/instructor with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT ensures that instructors have received an invite to the workshops and that PM team works with instructors to get setup in the platform (e.g. Teams, Teams Live Event)

## 1 Week Prior
- [ ] Connect vendor/instructor with the PM(s) for the locations where this event has been marketed
- [ ] Check-in with vendor/instructor that they are prepared for the workshop
- [ ] RCT moves the issue to the __Ready to Stream__ column

## During and After Stream
- [ ] Instructor (and optional moderator) run the stream
- [ ] _(Optional)_RCT submits recording to be uploaded to the Reactor YouTube and to be made available on LearnTV
- [ ] RCT moves issue to the __Stream Complete__ column

**Comment by KimberlyBolton1** [comment]
@meaghanlewis 
Edited abstract for your review:

Rust Programming Basics
Rust is a programming language that empowers developers to build reliable and efficient software. It is fun to write, easy to read, and high performing. In this workshop, we will explore how to start coding in Rust. This workshop will require you to use VS Code and the Rust extension to write code.

**Comment by meaghanlewis** [comment]
This edit looks good to me. ✅

**Comment by KimberlyBolton1** [comment]
@mebaumb
Abstract final and moved to next stage.

**Comment by mebaumb** [comment]
These titles/Abstract/Topics/Length of event updates have been added into Sharepoint. Waiting on DI for speakers (6/26)

**Comment by mebaumb** [comment]
Abu Dhabi, Bangalore and Tel Aviv have been published since we have Speaker info. (7/20)

**Comment by mebaumb** [comment]
Sent Sao Paulo title description to Caio for translation (7/23)

**Comment by mebaumb** [comment]
Need to publish Sao Paulo - Dynamics down

**Comment by mebaumb** [comment]
Stream numbers sent to DI and workshops delivered.
